conmon-rs/server/src/config.rs

```diff
@@ -88,6 +88,15 @@ pub struct Config {
     )]
     /// Do not fork if true
     skip_fork: bool,
+
+    #[get_copy = "pub"]
+    #[clap(
+        env(concat!(prefix!(), "SYSTEMD_CGROUP")),
+        long("systemd-cgroup"),
+        value_name("SYSTEMD_CGROUP")
```

Comment:
Thinking about this a bit more, do we want to use `cgroup-manager` here? it could give us more flexibility in the future, but also may be unnecessary

Reply:
Yep, done :+1: